tornado/test/httpserver_test.py

```diff
@@ -378,13 +379,28 @@ def check_type(self, name, obj, expected_type):
             self.errors[name] = "expected %s, got %s" % (expected_type, actual_type)
 
 
+class EchoUTF8Handler(RequestHandler):
```

Comment:
I'd name these classes `PostEchoHandler` and `PostEchoGBKHandler` (and the urls `/post_utf8` and `/post_gbk` to avoid confusion with the other `EchoHandler`.